### PR TITLE
Istvan

### PR DESCRIPTION
**Pull request by ghost**
done

### CODE REVIEW

File: src/CustomButton.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, {
+  Component
+} from 'react';
+import './App.css';
+
+class CustomButton extends Component {
+  render() {
+    return ( <button { ...this.props}>{this.props.id}</button>
```
Comment (dangroza) [inline src/CustomButton.js]: Please remove spaces between `(` and ` <` etc

File: src/GenericForm.js [hunk 2]
```diff
@@ -1,80 +1,144 @@
-import React, { Component } from 'react';
+import React, {
+  Component
```
Comment (dangroza) [inline src/GenericForm.js]: why on new line?

File: src/GenericForm.js [hunk 3]
```diff
@@ -1,80 +1,144 @@
...
+            let el = fields[key];
+            switch (el.type) {
+              case "text":
+                childNodes.push( < CustomInput { ...el
+                    }
+                    onChange = {
+                      this.handleFieldChange
+                    }
+                    />)
+                    break;
```
Comment (dangroza) [inline src/GenericForm.js]: there is a problem with the indentation here

**Review by dangroza — CHANGES_REQUESTED** [review]

File: src/GenericForm.js (hunk 3 above)
Comment (ghost) [reply src/GenericForm.js]: done

File: src/GenericForm.js (hunk 2 above)
Comment (ghost) [reply src/GenericForm.js]: done

File: src/CustomButton.js (hunk 1 above)
Comment (ghost) [reply src/CustomButton.js]: done

**Pull request merged**